### DISCUSSION TITLE
Question: Current recommended stable version for yocto/embedded linux

### DISCUSSION THREAD

**Discussion by tiny-dancer** [fluent/fluent-bit#8336] [Q&A] [answered]
Hey all, I see from the [docs](https://docs.fluentbit.io/manual/installation/yocto-embedded-linux) v1.8.11 from Dec 10, 2021 is the recommended stable version to use for yocto linux, i understand embedded linux may be older but this seems decently older, is this recommended stable version up to date?

Related: https://github.com/fluent/fluent-bit-docs/issues/1282

**Comment by patrick-stephens** [comment, marked as answer]
I think this is just a case of the docs not being updated. I would use the latest but be aware we have no real way to verify Yocto currently builds or functions in the Github CI so it is just a best effort support. 
Please submit any patches back to help improve.

**Reply by patrick-stephens** [reply]
If you try a newer version and it works then let's get the docs updated @tiny-dancer

**Reply by tiny-dancer** [reply]
@patrick-stephens this was exploratory for us - if we move forward will follow up here accordingly.
